charts/hono/values.yaml

```diff
@@ -182,6 +182,24 @@ useLoadBalancer: true
 # Configuration properties for protocol adapters.
 adapters:
 
+  # externalAdaptersEnabled indicates whether protocol adapters that
+  # run outside of the kubernetes cluster should be allowed to connect to
+  # the Dispatch Router and the example device registry's service endpoints.
+  # The default setting is 'false' which prevents access from outside of the
+  # cluster.
+  # Setting this property to 'true' allows external adapters to connect to
+  # the Dispatch Router's 'interal' endpoint and the Device Registry's
+  # service endpoints via AMQPS, i.e. AMQP over TLS.
+  # The Dispatch Router requires adapters to authenticate using SASL EXTERNAL,
+  # i.e. an adapter needs to provide a client certificate that has been signed
+  # by one of the CA certs contained in the router's trust store.
+  # When opening the AMQP connection to the router, the adapter needs to indicate
+  # the 'hono-internal' virtual host name in its AMQP 1.0 'open' frame.
```

Comment:
I think a hint would be helpful that the connection has to be opened to a different port than the one to be used by northbound applications when connecting to the Dispatch Router.